_includes/people.html

```diff
@@ -1,6 +1,6 @@
 <div class="card people-card" id="{{ username }}">
   <div class="card-content">
-    <div class="media">
+    <div class="media" style="height: 230px;">
```

Comment:
Can you remove the height here as it is in the CSS? And check everything if fine :smile:

Reply:
I could have sworn I removed it. How did I miss it? 😭